packages/go/dawgs/graph/types.go

```diff
@@ -16,7 +16,222 @@
 
 package graph
 
-import "github.com/specterops/bloodhound/dawgs/util/size"
+import (
+	"github.com/specterops/bloodhound/dawgs/cardinality"
+	"github.com/specterops/bloodhound/dawgs/util/size"
+	"sync"
+)
+
+type KindBitmaps map[string]cardinality.Duplex[uint64]
```

Comment:
Thank you for putting all of this in a more appropriate location 🙇

Reply:
:muscle: no broken windows